ci/tests.sh

```diff
@@ -4,5 +4,14 @@ set -e
 # -----
 # REPLACE THE BELOW WITH YOUR COMMANDS
 # -----
+echo 'Run Node tests'
+# Run Playwright tests
 npx playwright install --with-deps
 npx playwright test
+
+echo 'Run accessibility tests'
+# Run Cypress accessibility tests
+# Installing wait-on ensures the application is up and running before Cypress.
+npm install -g wait-on
+npm run serve & wait-on http://localhost:8080
```

Comment:
For pipeline check it makes a lot of sense for this to run in the background with no needed cleanup (because closing the container will kill all processes and calling).  Maybe a stretch, but where I see this potentially causing an error, is if someone were to be in the process of developing -> run this command with the server now running in the background -> test -> go back to development -> get distracted -> refresh localhost and everything looks good (but now this isn't including all development that needs restart to server, similarly if the accessibility test is run locally/docker it will not pickup changes).  Of course, this will all get flagged on the pipeline if there are issues.  If one were to rerun the server it will just increment the port and still ultimately point to 8080 (showing appropriate development changes).  Just maybe a reminder for people to kill their stale processes in the README or if we really think this could cause confusion you could add a kill process (one idea is `RUN apt-get install lsof` in Dockerfile and `kill $(lsof -t -i:8080)` here.  I think just a note in the Readme is sensible but I will leave the final decision up to you.

Reply:
Really like the idea of terminating the hanging process!

Reply:
@LynnHerrick9 This is a good case point. I like the idea of adding the kill process in, so I will go ahead and do that and add a note in the README as a reminder to developers. Thanks for the suggestion!